src/test/java/bio/terra/controller/RepositoryAPIControllerTest.java

```diff
@@ -0,0 +1,137 @@
+package bio.terra.controller;
+
+import bio.terra.JobMapKeys;
+import bio.terra.category.Unit;
+import bio.terra.model.JobModel;
+import bio.terra.model.StudySummaryModel;
+import bio.terra.stairway.FlightMap;
+import bio.terra.stairway.FlightState;
+import bio.terra.stairway.FlightStatus;
+import bio.terra.stairway.Stairway;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.sql.Timestamp;
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Optional;
+
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.Mockito.any;
+import static org.mockito.Mockito.when;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@AutoConfigureMockMvc
+@Category(Unit.class)
+public class RepositoryAPIControllerTest {
+
+    @MockBean
+    private Stairway stairway;
+
+    @Autowired
+    private MockMvc mvc;
+
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    private JobModel jobModel;
+
+    private static final String testFlightId = "test-flight-id";
+    private StudySummaryModel minimalStudySummary;
+
+    private static final Timestamp submittedTime = Timestamp.from(Instant.now());
+    private static final Timestamp completedTime = Timestamp.from(Instant.now());
```

Comment:
This will be two separate evaluations of Instant.now(), so the fractional seconds probably won't be the same.
If you want them the same, just do competedTime = submittedTime.